implementation/src/main/java/io/smallrye/jwt/auth/principal/KeyLocationResolver.java

```diff
@@ -349,4 +363,36 @@ public InputStream resolve(String keyLocation) throws IOException {
             return new URL(keyLocation).openStream();
         }
     }
+
+    boolean mayBePemKey() {
+        return isPemKey() || authContextInfo.getKeyFormat() == KeyFormat.ANY;
+    }
+
+    boolean isPemKey() {
+        return authContextInfo.getKeyFormat() == KeyFormat.PEM_KEY;
+    }
```

Comment:
What do you think about reducing the number of methods by using a parameter? E.g.

```java
boolean mayBeFormat(KeyFormat format) {
    return isFormat(format) || authContextInfo.getKeyFormat() == KeyFormat.ANY;
}

boolean isFormat(KeyFormat format) {
    return authContextInfo.getKeyFormat() == format;
}
```